bedrock/legal/urls.py

```diff
@@ -73,3 +69,16 @@
     ),
     path("defend-mozilla-trademarks/", views.fraud_report, name="legal.fraud-report"),
 )
+
+if switch("firefox-tou"):
```

Comment:
Style nit: Should this be `FIREFOX_TOU` to remain in style with the other switches?